blimp-live-widgets/src/components/Clock/index.js

```diff
@@ -46,8 +47,8 @@ export default class Clock extends Component {
   render() {
 
     return (
-      <div>
-        <ClockLayout {...this.state} styles={this.styles} showSmallTicks={this.props.showSmallTicks} />
```

Comment:
Idk whether we should remove or comment this out. I asked on the chat what people want to do - please just follow that convo.

Reply:
Addressed